solutions/observability/apm/transaction-sampling.md

```diff
@@ -290,6 +290,46 @@ This example defines three tail-based sampling polices:
 2. Samples 1% of traces in `production` with the trace name `"GET /not_important_route"`
 3. Default policy to sample all remaining traces at 10%, e.g. traces in a different environment, like `dev`, or traces with any other name
 
+### Example configuration 2 [_example_configuration_2]
+
+When a trace originates in Service A and then calls Service B, the sampling rate is determined by the service where the trace starts:
+
+```yaml
+- sample_rate: 0.3
+  service.name: B
+- sample_rate: 0.5
+  service.name: A
+- sample_rate: 1.0  # Fallback: always set a default
+```
+
+- Because Service A is the root of the trace, its policy (0.5) is applied while Service B's policy (0.3) is ignored.
+- If instead the trace began in Service B (and then passed to Service A), the policy for Service B would apply.
+
+> **Key point**: Tail‑based sampling rules are evaluated at the *trace level* based on which service initiated the distributed trace, not the service of the transaction or span.
+
+### Example configuration 3 [_example_configuration_3]
+
+Policies are evaluated **in order** and applies the first one whose match conditions are all met. That means, in practice, order policies from most specific (narrow matchers) to most general, ending with a catch-all (fallback).
+
+```yaml
+# Example A: prioritize service origin, then failures
+- sample_rate: 0.2
+  service.name: A
+- sample_rate: 0.5
+  trace.outcome: failure
+- sample_rate: 1.0  # catch-all
+
+# Example B: prioritize failures, then a specific service
+- sample_rate: 0.2
+  trace.outcome: failure
+- sample_rate: 0.5
+  service.name: alice
```

Comment:
did you mean service A instead of alice, if you're trying to illustrate that the first policy will apply?